docs/AuthRBAC.md

```diff
@@ -850,9 +850,9 @@ const ProductShow = () => (
 );
 ```
 
-## `<Tab>`
+## `<TabbedShowLayout.Tab>`
```

Comment:
You should rename this section to `<TabbedShowLayout>`, and copy the docs and code from https://marmelab.com/ra-enterprise/modules/ra-rbac#tabbedshowlayout.

No need for a `<TabbedShowLayout.Tab>` section IMO as it would go into too much detail (and is already mentioned in the code snippets).